pkg/securitycontextconstraints/sccmatching/provider.go

```diff
@@ -460,8 +456,8 @@ func createCapabilitiesStrategy(defaultAddCaps, requiredDropCaps, allowedCaps []
 }
 
 // createSeccompStrategy creates a new seccomp strategy
-func createSeccompStrategy(allowedProfiles []string) (seccomp.SeccompStrategy, error) {
-	return seccomp.NewWithSeccompProfile(allowedProfiles)
+func createSeccompStrategy(scc *securityv1.SecurityContextConstraints) (seccomp.SeccompStrategy, error) {
+	return seccomp.NewSeccompStrategy(scc.SeccompProfiles), nil
```

Comment:
this was just cosmetic?

Reply:
it was a bit of a different change as I was trying to get closer to upstream, I can turn it back

Reply:
reverted back to previous state